### ISSUE TITLE
Build fail since FAI upgraded to 6.2

### ISSUE THREAD

**Issue by TanguyRaufflet** [seapath/build_debian_iso#93]
Impossible to build an image for VM. It seems to be linked with the upgrade of FAI and a package that failed at the installation. I manage to build with the the version 5376150.

```
The following held packages will be changed:
  linuxlogo nullmailer unattended-upgrades
0 upgraded, 211 newly installed, 0 to remove and 0 not upgraded.
E: Held packages were changed and -y was used without --allow-change-held-packages.
ERROR: 25600 25600
ERROR: chroot /tmp/fai-diskimage.M5xY42 apt-get -y -o Dpkg::Options::=--force-confdef -o Dpkg::Options::=--force-confnew --fix-missing install --no-install-recommends  bash-completion debconf-utils file zstd less linuxlogo rsync openssh-client openssh-server time procinfo nullmailer sudo locales console-setup kbd pciutils usbutils unattended-upgrades grub-efi dosfstools amd64-microcode apparmor-utils at audispd-plugins auditd bridge-utils ca-certificates chrony cpuset curl docker-compose docker.io dstat efibootmgr gnupg grub-efi-amd64-signed intel-microcode iptables-persistent irqbalance jq lbzip2 linux-image-rt-amd64/bookworm-backports linuxptp lm-sensors lsb-release lvm2 net-tools openssh-server ovmf parted python3-apt python3-cffi-backend python3-setuptools shim-signed sudo sysfsutils syslog-ng sysstat systemd-resolved vim wget nullmailer- linuxlogo- unattended-upgrades- memtest86+- dbus return code 100
install_packages: executing chroot /tmp/fai-diskimage.M5xY42 apt-get clean
install_packages: executing chroot /tmp/fai-diskimage.M5xY42 apt-get -y -o Dpkg::Options::=--force-confdef -o Dpkg::Options::=--force-confnew --purge remove isc-dhcp-client isc-dhcp-common
Reading package lists...
Building dependency tree...
The following packages will be REMOVED:
  isc-dhcp-client* isc-dhcp-common*
0 upgraded, 0 newly installed, 2 to remove and 0 not upgraded.
After this operation, 3050 kB disk space will be freed.
(Reading database ... 8745 files and directories currently installed.)
Removing isc-dhcp-client (4.4.3-P1-2) ...
Removing isc-dhcp-common (4.4.3-P1-2) ...
(Reading database ... 8721 files and directories currently installed.)
Purging configuration files for isc-dhcp-client (4.4.3-P1-2) ...
install_packages: executing chroot /tmp/fai-diskimage.M5xY42 dpkg --configure --pending
install_packages: executing chroot /tmp/fai-diskimage.M5xY42 dpkg -C
install_packages: executing chroot /tmp/fai-diskimage.M5xY42 apt-get clean
1 errors during executing of install_packages
E: Held packages were changed and -y was used without --allow-change-held-packages.
Exit code task_instsoft: 472
Calling task_configure
Executing shell: DEBIAN/10-rootpw
DEBIAN/10-rootpw     OK.
Executing shell: DEBIAN/20-capabilities
DEBIAN/20-capabilities OK.
Executing shell: DEBIAN/30-interface
DEBIAN/30-interface  OK.
Executing shell: DEBIAN/40-misc
DEBIAN/40-misc       OK.
Executing shell: FAIBASE/10-misc
FAIBASE/10-misc      OK.
Executing shell: FAIBASE/15-root-ssh-key
FAIBASE/15-root-ssh-key OK.
Executing shell: FAIBASE/20-removable_media
FAIBASE/20-removable_media OK.
Executing shell: SEAPATH_COMMON/10-rootpw
SEAPATH_COMMON/10-rootpw FAILED with exit code 2.
Executing shell: SEAPATH_COMMON/20-grub_default
SEAPATH_COMMON/20-grub_default FAILED with exit code 127.
Executing shell: SEAPATH_COMMON/30-change_editor
SEAPATH_COMMON/30-change_editor OK.
Executing shell: SEAPATH_COMMON/40-networking
SEAPATH_COMMON/40-networking OK.
Executing shell: SEAPATH_COMMON/50-aptrepos
SEAPATH_COMMON/50-aptrepos OK.
Executing shell: SEAPATH_COMMON/60-misc
SEAPATH_COMMON/60-misc OK.
Executing shell: GRUB_EFI/10-setup
GRUB_EFI/10-setup    FAILED with exit code 127.
Executing shell: LAST/50-misc
LAST/50-misc         FAILED with exit code 1.
Exit code task_configure: 420
```

**Comment by insatomcat** [comment]
I can't reproduce.
Before going deeper, can you please confirm:
- that you are using a building host with UEFI ?
- that you rebuild the fai docker image recently ?
Thanks.

**Comment by TanguyRaufflet** [comment]
Yes i'm using a building host with UEFI since I have the folder /sys/firmware/efi on it. Concerning the image, I've use the command ``` docker rmi fai -f ``` before the build. Erwann also tried on his PC and got the same issue.

**Comment by insatomcat** [comment]
Can you send the full log please?

**Comment by insatomcat** [comment]
can you please try with branch "issue93"?

**Comment by yairpod** [comment]
@insatomcat , Having encountered the same issue, branch "issue93" solved it for me.

**Comment by insatomcat** [comment]
solved by #94
